src/exchange/swap/sortProvidersByWeight.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import type { AvailableProvider } from "./types";
+
+const providerPriorityMap = { changelly: 1 };
```

Comment:
Wanted to move this map to a JSON file but apparently *.json files aren't included in the post-build bundle. Tell me if I missed something, will be happy to move this data to allow isolate updating. 

Plus we could imagine moving these priorities from here to the CMS, It's a simple idea.